jest.config.js

```diff
@@ -1,4 +1,4 @@
-'use strict';
+
```

Comment:
Any reason removing the `use strict`? I believe if we remove this, we don't need the ln#1 and 2 anymore.

Reply:
Oops, fixed it now. I need to adjust the eslint config so it doesn't autofix it away.